### PR TITLE
NCSD-2302: Composite registration verbose logging

### PR DESCRIPTION
**Pull request by rivoric**
Added additional logging for POST and PATCH commands so that the body is outputted. This will allow debugged of issues around registration.json

### CODE REVIEW

File: PSModules/CompositeRegistrationFunctions.psm1 [hunk 1]
```diff
@@ -227,6 +232,7 @@ New-PathRegistration -Path $pathObject
     return Invoke-CompositeApiRegistrationRequest -Url $finalUrl -Method Post -RequestBody $requestBodyText
 }
 
+[CmdletBinding]
```
Comment (NickGraham101) [inline PSModules/CompositeRegistrationFunctions.psm1]: Aren't the brackets required in [CmdletBinding()]

File: PSModules/CompositeRegistrationFunctions.psm1 [hunk 2]
```diff
@@ -282,6 +288,7 @@ New-RegionRegistration -Path somePath -Region $regionObject
     return Invoke-CompositeApiRegistrationRequest -Url $finalUrl -Method Post -RequestBody $requestBodyText    
 }
 
+[CmdletBinding]
```
Comment (NickGraham101) [inline PSModules/CompositeRegistrationFunctions.psm1]: Aren't the brackets required in [CmdletBinding()]

File: PSModules/CompositeRegistrationFunctions.psm1 [hunk 3]
```diff
@@ -336,6 +343,7 @@ Update-PathRegistration -Path somePath -ItemsToUpdate $itemsToUpdate
     return Invoke-CompositeApiRegistrationRequest -Url $finalUrl -Method Patch -RequestBody $requestBodyText
 }
 
+[CmdletBinding]
```
Comment (NickGraham101) [inline PSModules/CompositeRegistrationFunctions.psm1]: Aren't the brackets required in [CmdletBinding()]

**Comment by rivoric** [comment]
I've removed all [CmdletBinding] lines, the verbose logging works without them. CmdletBinding must only be needed on the top most function / script

**Pull request merged**